clients/cobol-lsp-vscode-extension/src/services/util/Utils.ts

```diff
@@ -40,4 +41,11 @@ export class Utils {
     await ext.activate();
     return ext.exports as any;
   }
+  public static setExtensionFolder(path: string) {
+    this.extPath = path;
+  }
+
+  public static getExtensionFolder(): string {
+    return this.extPath;
+  }
```

Comment:
I still think this should be removed in favour of extra parameters on affected functions.

Reply:
any Idea how to get global storage path outside of activation context ?

Reply:
what do you mean? You are getting it out of the context right now, so just pass it as parameters to the various services on construction and functions as needed.

Reply:
A static value can be used throughout extension. This is never a changing value, it is not needed to build a dependency injection mechanism for this.

Reply:
You are injecting the value anyway, it is just harder to see and needs to be mocked for testing.

Reply:
Services that consumes this e.g CopybookDownloadService methods are static methods. If I pass it on creation, all of these methods / tests needs to be refactored.

Reply:
I am aware.

Reply:
refactored at : 533bc9f181e4bd6b8c63959229b92a9797bcbb99